gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/architecture/CoreRulesTest.java

```diff
@@ -42,6 +42,7 @@ public void core_should_be_independent() {
                     "org.slf4j..",
                     "lombok..",
                     "com.fasterxml..",
+                    "com.google..",
```

Comment:
We should carefully think what kind of exception we add here. We don't really need the VisibleForTesting annotation so I won't add that exception